sat/cli/bootsys/mgmt_power.py

```diff
@@ -239,13 +240,20 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
         LOGGER.warning('Forcibly powering off nodes: %s', ', '.join(pending_hosts))
 
         # Confirm all nodes have actually turned off.
-        failed_hosts = IPMIPowerStateWaiter(pending_hosts, 'off', ipmi_timeout, username, password,
-                                            send_command=True).wait_for_completion()
+        prompt_message = 'Do you want to forcibly power off the nodes after timeout?'
```

Comment:
minor: I think "after timeout" is confusing here. It sounds like we're going to wait for another timeout and then power off the nodes, but in reality, we are going to power them off immediately. Either leave off the mention of "after timeout" or say this:

```suggestion
        prompt_message = 'Do you want to forcibly power off the nodes that timed out?'
```